src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <main>
+        <div class="left-main">
```

Comment:
- [ ] The `<h1>Leaderboard</h1>` should be enclosed in a `<header>` tag as it is essentially the header  of your document 👍